examples/chemistry/qchem_modelling_basics.ipynb

```diff
@@ -270,7 +280,7 @@
     "id": "sd8GEceHMfWB"
    },
    "source": [
-    "We see in the previous graph that increasing the basis complexity brings the energy eigenvalue closer to the true Hartree-Fock energy. This trend can be generalized to other methods as well, like [CCSD](https://en.wikipedia.org/wiki/Coupled_cluster), [MP2](https://en.wikipedia.org/wiki/M%C3%B8ller%E2%80%93Plesset_perturbation_theory), [FCI](https://en.wikipedia.org/wiki/Full_configuration_interaction), (...) although the time to solution would be greater for those post Hatree-Fock approaches. The takeaway is that even if an infite basis is required to converge to the true energy, there are diminishing returns at some point, especially when considering the time required for the calculation. In our case, `6-311G(d,p)` appears as a good choice in term of accuracy and compute time.\n",
+    "We see in the previous graph that increasing the basis complexity brings the energy eigenvalue closer to the true Hartree-Fock energy. This trend can be generalized to other methods as well, like [CCSD](https://en.wikipedia.org/wiki/Coupled_cluster), [MP2](https://en.wikipedia.org/wiki/M%C3%B8ller%E2%80%93Plesset_perturbation_theory), [FCI](https://en.wikipedia.org/wiki/Full_configuration_interaction), (...) although the time to solution would be greater for those post Hartree-Fock approaches. The takeaway is that even if an infinite basis is required to converge to the true energy, there are diminishing returns at some point, especially when considering the time required for the calculation. In our case, `6-311G(d,p)` appears as a good choice in term of accuracy and compute time.\n",
     "\n",
     "## Consideration of Correlation Energy - Potential of Quantum Computing\n",
```

Comment:
The introduction of correlation energy for me was not very clear. `e` is a computed energy but by what ? 

Maybe we can introduce `e` as the exact theoretical energy (e_corr + e_HF) and then  explain in 1-2 sentences that in practice post Hartree-Fock methods attempt to recover as much as the correlation energy as possible, in different ways (you can go back to talking about CCSD or anything you've mentioned already as an example), and that this is challenging. The more correlation energy they successfully recover, the closer e_corr + e_HF is to the exact energy (which is lower).

Please feel free to adapt this in any way that makes more sense chemically speaking, this is just the way i see it as a non-expert.

Reply:
I modified the paragraph to say
```
Where $E$ and $E_{\text{HF}}$ are the energies computed from the time-independent Schrödinger equation with a given Hamiltonian and the Hartree-Fock method, respectively. This Hamiltonian can exhibit a variety of arrangements, each with varying levels of complexity, depending on the post Hartree-Fock method used. Typically, more detailed ones aim to recover as much of the correlation energy as possible, approaching the exact Born-Oppenheimer energy, which corresponds to the full diagonalization of the Hamiltonian expressed in matrix form.
```

- Changing epsilon to E makes a link with the Schrödinger equation (first eq in the notebook).
- I introduced that there are many PHF methods with different complexities. I felt that the explanation about the "challenging" part was already explain in the next 2 paragraphs, so I left this point there.

Reply:
Thank you ! I'm suggesting we run this specific part by @LeeH4t1qbit to see if this is the best sentences we can craft.

Reply:
"in which E and E_{HF} are the energies computed from exact diagonalization of the molecular Hamiltonian within the Born-Oppenheimer approximation (i.e. Full Configuration Interaction (FCI)) and the Hartree-Fock method, respectively. Since it is computationally prohibitive to solve the FCI problem, with the exception of very small systems, it is typical to introduce approximations to include correlation effects in electronic structure calculations. The FCI problem considers the diagonalization of the Hamiltonian in a basis of all possible excited determinants, that is all possible 1-, 2-, 3-, …, N-body determinants for an N-electron system. Hence, the FCI wavefunction is a linear combination of all possible determinants for an N-electron system.  Approximate methods, only include a subset of the excited determinants in the approximate wavefunction in order to capture the most important correlation effects (e.g. many-body perturbation theory, coupled-cluster theory)."

This is the new suggestion, based on Lee's insights.